index.html

```diff
@@ -152,14 +152,27 @@ <h2>About Myself</h2>
                         </div>
                     </div>
                 </div>
-                <div class="about-languages">
-                    <img class="iconss" src="img2/angle-right.png" alt="vector">
+                <div class="about-languages bor-bot">
+                    <img  class="iconss" src="img2/angle-right.png" alt="vector">
                     <P>Frameworks</P>
                 </div>
-                <div class="about-languages">
+                <div class="about-languages bor-bot">
                     <img class="iconss" src="img2/angle-right.png" alt="">
                     <P>Skills</P>
                 </div>
+                <footer class="footer-container">
+                    <form action="https://formspree.io/f/mayzvjww" method="post" id="form">
+                        <h2>Contact me</h2>
+                        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs
+                        coding.
+                        I’d love to help with it!</p>
+                        <input name="username" type="username" placeholder="lorem lpsum" maxlength="30" required>
+                        <input name="email" type="email" placeholder="loremipsum@gmail.com" required>
+                        <textarea name="textarea" maxlength="500" placeholder="write your message here.."></textarea>
+                        <button name="button" type="submit" id="btn-f">Get in touch</button>
+                        <span></span>
+                    </form>    
+                </footer>
```

Comment:
but kindly explain which break-point you mean, the mobile? or the 768 desktop view port?,  because the one you just provided is not in Figma and I can't find it so kindly give some explanation @tresorsawasawa